types/LICENSE

```diff
@@ -0,0 +1,21 @@
+    MIT License
+
+    Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
copyright microsoft?

Reply:
well, it does have to be MIT: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/LICENSE

but it's not (c) microsoft